src/repositories/soft-crud.repository.base.ts

```diff
@@ -154,10 +166,12 @@ export abstract class SoftCrudRepository<
       filter = filter ?? {};
       filter.where = {
         deleted: false,
-        id: id,
       } as Condition<T>;
     }
 
+    //adding primary key property in where filter
+    filter.where[pk] = id;
```

Comment:
For cases where and/or operators are used?

Reply:
done